frontend/api/admin.ts

```diff
@@ -27,13 +27,30 @@ export class Admin {
    *
    * @param {string} name
    * @param {string} email
+   * @param {string} superUser
    */
-  constructor(public name: string, public email: string) {
+  constructor(
+    public name: string,
+    public email: string,
+    public superUser: boolean = false
+  ) {
     if (email.indexOf(":") != -1) {
       throw new EmailColonError(`${email} has an unsupported character ':'`);
     }
```

Comment:
This is actually no longer necessary, I'll remove it. It was really only applicable if we were sending user credentials via the Authorization header. Having `test:@gmail.com:password` would've been a major bug, but we don't do that so this can be removed.